### DISCUSSION TITLE
Imputation when comparing multiome gex and atac?

### DISCUSSION THREAD

**Discussion by mitchTwoTimes** [GreenleafLab/ArchR#1544] [Questions / Documentation] [answered]
Greetings,

I've struggled to find a definitive answer on whether I should impute the gene score when comparing to gene expression. For example, in the attached plot where the points are cells, the y-axis is expression for a single gene, and the x-axis is the matched accessibility, does it make sense to impute? FWIW the spearman's rho gets a little better with imputation.

<img width="259" alt="image" src="https://user-images.githubusercontent.com/92815743/182672046-b2826087-56e5-480f-a706-01507edb7d4d.png">

Thank you

**Comment by rcorces** [comment, marked as answer]
I'm not sure I have a hard answer for this. I think its fine to impute though I could see this being disputed. Certainly it seems safest to not impute if it looks ok without imputation.
